packages/devnet/run.sh

```diff
@@ -5,6 +5,7 @@ mkdir -p /var/log
 
 # Start Geth and direct output to stdout
 /geth \
+    --allow-insecure-unlock \
```

Comment:
Due to a new security feature added in the latest version of Geth, you can't unlock accounts and enable the RPC API without this flag.